### PR TITLE
Core Unit tests for Budget #4781

### LINKED ISSUE #4781: Core Unit tests for Budget
- [x] occ/adapters/organization/converters/occ-budget-normalizer.spec.ts
- [X] occ/adapters/organization/occ-budget.adapter.spec.ts
- [X] organization/connectors/budget/budget.connector.spec.ts 
- [X] organization/store/actions/budget.action.spec.ts 
~organization/store/reducers~
- [x] organization/store/effects/budget.effect.spec.ts
- [X] organization/store/selectors/budget.selectors.spec.ts 
- [x] organization/facade/budget.service.spec.ts

### PR DESCRIPTION
**Pull request by Pucek9**
Closes #4781

### CODE REVIEW

**Comment by Pucek9** [comment]
image

File: projects/core/src/organization/connectors/budget/budget.connector.spec.ts [hunk 1]
```diff
@@ -5,25 +5,57 @@ import { BudgetAdapter } from './budget.adapter';
...
   it('should be created', () => {
     expect(service).toBeTruthy();
   });
 
+  it('should load budget', () => {
+    service.get(userId, budgetCode);
+    expect(adapter.load).toHaveBeenCalledWith(userId, budgetCode);
+  });
+
+  it('should load budgets', () => {
```
Comment (marlass) [inline projects/core/src/organization/connectors/budget/budget.connector.spec.ts]: Could you check if provided params are passed?

File: projects/core/src/organization/store/effects/budget.effect.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export class BudgetEffects {
   );
 
   @Effect()
-  $loadBudgets: Observable<
+  loadBudgets$: Observable<
```
Comment (marlass) [inline projects/core/src/organization/store/effects/budget.effect.ts]: Why LoadBudgetSuccess is imported directly not through namespace?

File: projects/core/src/organization/store/effects/budget.effect.spec.ts [hunk 3]
```diff
@@ -78,16 +86,102 @@ describe('Budget Effects', () => {
...
+      expected = cold('-b', { b: completion });
+
+      expect(effects.loadBudget$).toBeObservable(expected);
+      expect(budgetConnector.get).toHaveBeenCalledWith(userId, budgetCode);
+    });
+
+    it('should return LoadBudgetFail action if budget not updated', () => {
+      budgetConnector.get = createSpy().and.returnValue(throwError(error));
+      const action = new BudgetActions.LoadBudget({ userId, budgetCode });
+      const completion = new BudgetActions.LoadBudgetFail(error);
```
Comment (marlass) [inline projects/core/src/organization/store/effects/budget.effect.spec.ts]: I think you need code in payload here. Is it provided in error returned from API?
Applies to every Fail action

File: projects/core/src/organization/store/selectors/budget.selectors.spec.ts [hunk 4]
```diff
@@ -1,124 +1,95 @@
...
 
-    store = TestBed.get(Store as Type<Store<StateWithProduct>>);
+    store = TestBed.get(Store as Type<Store<StateWithOrganization>>);
     spyOn(store, 'dispatch').and.callThrough();
   });
 
-  describe('getSelectedProductsFactory', () => {
-    it('should return product by code', () => {
-      let result: Product[];
+  describe('getBudgetManagmentState ', () => {
```
Comment (marlass) [inline projects/core/src/organization/store/selectors/budget.selectors.spec.ts]: Typo. Could you fix in every place `Managment` to `Management`?

File: projects/core/src/organization/store/selectors/budget.selectors.spec.ts [hunk 5]
```diff
@@ -1,124 +1,95 @@
...
-      let result: Product;
       store
-        .pipe(select(ProductSelectors.getSelectedProductFactory(code)))
+        .pipe(select(BudgetSelectors.getBudgetsState))
         .subscribe(value => (result = value));
 
-      store.dispatch(new ProductActions.LoadProductSuccess(product));
-      expect(result).toEqual(product);
+      store.dispatch(new BudgetActions.LoadBudgetSuccess([budget, budget2]));
+      // @ts-ignore
```
Comment (marlass) [inline projects/core/src/organization/store/selectors/budget.selectors.spec.ts]: Do we need it?

File: projects/assets/src/translations/en/user.ts [hunk 6]
```diff
@@ -51,7 +51,7 @@ export const user = {
       label: 'Confirm password',
       placeholder: 'Confirm Password',
     },
-    managmentInMyAccount: 'Management in My Account.',
+    ManagementInMyAccount: 'Management in My Account.',
```
Comment (marlass) [inline projects/assets/src/translations/en/user.ts]: camelCase?

**Pull request merged**